### DISCUSSION TITLE
Sethome Permission

### DISCUSSION THREAD

**Discussion by ItLucifer-MRPG** [Zrips/CMI#6082] [Feature Requests]
Hi, I love how simple the sethome feature is, however I have temporary bought ranks on my server, I would love to be able to have a permission like "cmi.command.extrasethome.2" where it adds the sethomes to the existing cmi.command.sethome.[number] so I sell and give players homes that will stay consistently as they change and lose ranks

**Comment by dan28000** [comment]
https://www.zrips.net/cmi/commands/specialized/
Look at the ptarget example
Im kinda confused from your question what you want to do exactly

**Reply by ItLucifer-MRPG** [reply]
Maybe it's possible and I just don't understand it, but what I'm asking for is a permission to add a way to add perms of set homes together. Right now if I set a person to have cmi.command.sethome.5 and the group they're in the have cmi.command.sethome.2 instead of giving them 7 homes, it will give them 5 and override the 2. As I plan on having a store they they can buy set homes, which will be pointless if they buy two homes but their rank has 5 so they just stay at 5.
